src/eval.c

```diff
@@ -596,6 +596,8 @@ typval2string(typval_T *tv, int convert)
 	ga_append(&ga, NUL);
 	retval = (char_u *)ga.ga_data;
     }
+    else if (convert && tv->v_type == VAR_DICT)
```

Comment:
The comment above this function should need update.

Reply:
Thanks.  Updated the comment to mention that a Dict is converted into its textual representation.